### ISSUE TITLE
Version 6.0 source code not available

### ISSUE THREAD

**Issue by deltecent** [ssshake/esp8266_modem#38]
It would be very nice if the source code for Version 6 hardware was available.

**Comment by veremenko-y** [comment]
Yes please! After doing OTA update I now get filesystem errors, which I can't even track down without source code.

**Comment by synthead** [comment]
Amy update on this one?

**Comment by deltecent** [comment]
I am starting to think the thing to do is fork this repository and start updating ourselves for Version 6 hardware. It appears that @ssshake has basically abandoned this project. I have a schematic made for the hardware. We can post the KiCAD and Gerber files in the repository too so people can build their own board. I chose to use through-hole components so they are easy to assemble from a bare PCB.
